users/README.md

```diff
@@ -34,6 +34,12 @@
 		"super_user": "boolean",
 		"in_discord": "boolean"
 	},
+    "archivedDetails": {
+    "archived_at": "String",
+    "archived_user_id": "String",
```

Comment:
we can call this this `userId`

Reply:
In fact the this whole object belongs to the userid we can remove the `userId` itself from here as it is present at the top.